### DISCUSSION TITLE
outdated section heading in "install" section of doc

### DISCUSSION THREAD

**Discussion by goyalyashpal** [qutebrowser/qutebrowser#8136] [Q&A]
hi, 

is it deliberate or an oversight that this section heading has name of "Chocolatey package" whereas, it now shows 3 different ways to get it, the other 2 non-chocolatey ones being scoop & powershell's inbuilt funciton

https://www.qutebrowser.org/doc/install.html#_a_href_https_chocolatey_org_packages_qutebrowser_chocolatey_package_a

![image](https://github.com/qutebrowser/qutebrowser/assets/19423063/80a263f0-36f6-463c-b306-fd8447545b16)

https://github.com/qutebrowser/qutebrowser/blob/c073412b49002ff75bf67fac1c3e59560135a51b/doc/install.asciidoc?plain=1#L283-L298

**Comment by The-Compiler** [comment]
Good catch, thanks! Fixed in 93b8438eb.
